### DISCUSSION TITLE
Fedora 41 - Gnome 47 support

### DISCUSSION THREAD

**Discussion by Kusho50** [home-sweet-gnome/dash-to-panel#2174] [General]
Trying to install this to extensions in Fedora 41 Workstation beta (Gnome 47), but it says it is not compatible.  Is there an update planned?  Or am I doing something wrong to get it working on Fedora 41?

**Comment by Kusho50** [comment]
![Screenshot From 2024-09-16 13-04-34](https://github.com/user-attachments/assets/f32adeaf-db96-488f-8d92-a12f58ff9412)

**Reply by psarraf** [reply]
ArchLinux has an updated version that works for Gnome 47. It worked for me in Fedora 41 as well.
[https://archlinux.org/packages/extra/any/gnome-shell-extension-dash-to-panel/](url)

**Reply by Kusho50** [reply]
Went to that site, but didn't see anywhere to download it...  Did find that package on dnf, but after I installed it, nothing changed, and it didn't add it to Extensions, or Extension Manager...  didn't see any new extension manager installed..  so how do I use it?

tom@fedora:~$ sudo dnf install gnome-shell-extension-dash-to-panel
[sudo] password for tom: 
Updating and loading repositories:
 Fedora 41 - x86_64                                                                                      100% |  46.7 KiB/s |  26.0 KiB |  00m01s
 Fedora 41 - x86_64 - Updates                                                                            100% | 110.8 KiB/s |  28.5 KiB |  00m00s
 Fedora 41 - x86_64 - Test Updates                                                                       100% |  47.1 KiB/s |  12.5 KiB |  00m00s
 Fedora 41 - x86_64 - Test Updates                                                                       100% |   1.0 MiB/s |   1.2 MiB |  00m01s
Repositories loaded.
Package                                         Arch         Version                                          Repository                     Size
Installing:
 gnome-shell-extension-dash-to-panel            noarch       62-3.fc41                                        fedora                      1.1 MiB

Transaction Summary:
 Installing:        1 packages

Total size of inbound packages is 225 KiB. Need to download 225 KiB.
After this operation 1 MiB will be used (install 1 MiB, remove 0 B).
Is this ok [y/N]: y
[1/1] gnome-shell-extension-dash-to-panel-0:62-3.fc41.noarch                                             100% | 465.8 KiB/s | 225.5 KiB |  00m00s
-------------------------------------------------------------------------------------------------------------------------------------------------
[1/1] Total                                                                                              100% | 295.9 KiB/s | 225.5 KiB |  00m01s
Running transaction
[1/3] Verify package files                                                                               100% | 333.0   B/s |   1.0   B |  00m00s
[2/3] Prepare transaction                                                                                100% |   6.0   B/s |   1.0   B |  00m00s
[3/3] Installing gnome-shell-extension-dash-to-panel-0:62-3.fc41.noarch                                  100% |   3.6 MiB/s |   1.1 MiB |  00m00s
>>> Running trigger-install scriptlet: glib2-0:2.82.0-1.fc41.x86_64
>>> Stop trigger-install scriptlet: glib2-0:2.82.0-1.fc41.x86_64
>>> Running trigger-install scriptlet: glib2-0:2.82.0-1.fc41.i686
>>> Stop trigger-install scriptlet: glib2-0:2.82.0-1.fc41.i686
Complete!
tom@fedora:~$

**Reply by Kusho50** [reply]
also tried gnome-shell-extension-tool

tom@fedora:~$ gnome-shell-extension-tool -e dash-to-panel
gnome-shell-extension-tool is deprecated, use gnome-extensions instead

so tried gnome-extensions

but it seemed to say gnome-shell-extension-dash-to-panel doesn't exist, no such file or directory.  Searched the internet but not seeing any way to use.  Need some help to get this working, not sure the syntax.

**Reply by Kusho50** [reply]
https://extensions.gnome.org/review/31647

Installed and it works on Fedora 41 - Gnome 47

Seems this has been out for 2 weeks.  Thanks for the lack of help.
